NEWS.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+## [7.5.0] - 2016-08-03
+
+### Added
+
+* Support the new `electron` package name. #435
```

Comment:
The convention is to avoid periods at the end of items and put GitHub issue numbers in parentheses.